proto/workload.proto

```diff
@@ -74,6 +74,36 @@ message Service {
   // Waypoint is the waypoint proxy for this service. When set, all incoming requests must go
   // through the waypoint.
   GatewayAddress waypoint = 7;
+
+  // Load balancing policy for selecting endpoints.
+  // Note: this applies only to connecting directly to the workload; when waypoints are used, the waypoint's load_balancing
+  // configuration is used.
+  LoadBalancing load_balancing = 8;
+}
+
+message LoadBalancing {
+  enum Targets {
+    UNSPECIFIED = 0;
```

Comment:
would ANY or NONE read more cleanly in docs.

"When the locality_preference is set to [NONE] traffic can be routed to any endpoint that is a member of the mesh"